src/ngx_http_lua_common.h

```diff
@@ -22,6 +22,19 @@
 #include <lauxlib.h>
 
 
+#if (NGX_PCRE)
+
+#include <pcre.h>
+
+#if (PCRE_MAJOR > 8) || (PCRE_MAJOR == 8 && PCRE_MINOR >= 21)
+#   define LUA_HAVE_PCRE_JIT 1
+#else
+#   define LUA_HAVE_PCRE_JIT 0
+#endif
+
+#endif
```

Comment:
Now that we define these macros in this `common.h` header that is included in all the other compilation units (directly or indirectly), we should remove the duplicate definitions from `regex.c` I think?